packages/react-vapor/src/components/calendar/Calendar.tsx

```diff
@@ -58,7 +58,7 @@ export const DEFAULT_MONTHS: string[] = moment.months();
 export const DEFAULT_YEARS: string[] = [
     ...DateUtils.getPreviousYears(10),
     DateUtils.currentYear.toString(),
-    ...DateUtils.getNextYears(10),
+    ...DateUtils.getNextYears(30),
```

Comment:
what is this doing and why is this changed to `30` while the `getPreviousYears` hasn't change ?

Reply:
It extends the future years available to 30 years instead of 10 since the original issue saw a date come from the API of 2038. 10 years in the future wasn't enough but 50 feels like overkill. 30 was the middle ground. I don't know know why I didn't change the previous years but I guess it won't hurt